PhilipFunkos/src/App.css

```diff
@@ -0,0 +1,18 @@
+#root {
+  max-width: 1280px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 0;
+  padding: 2rem;
```

Comment:
Do not use rem for paddings, only use rem for things related to font, like font-size and line height. Use pixels for everything else.